test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -300,6 +300,44 @@ public void EntryPointCatalog()
             Done();
         }
 
+        [Fact]
+        public void EntryPointCatalogCheckDuplicateParams()
+        {
+            // Run this test to prevent introducing duplicate param names in entrypoints
+            // TODO: fix entrypoints in excludeSet from having duplicate param names
+            var excludeSet = new HashSet<string>();
```

Comment:
>excludeSet [](start = 16, length = 10)

Should we fix these as well as part of this PR ?  

Or is there a separate issue created to handle these separately ?

Reply:
Pls see the description of the PR where I suggest to fix each component in separate PR. Firstly this will keep PRs small and focused on specific component. Secondly to get consensus on name need to talk to original author. I ll open issues on each of it.

Reply:
I prefer to fix each component in separate PR. Firstly this will keep PRs small and focused on specific component. Secondly to get consensus on name one needs to talk to original author. I can open issues on each of it. 

---
In reply to: [223161784](https://github.com/dotnet/machinelearning/pull/1169#discussion_r223161784) [](ancestors = 223161784)